flags.go

```diff
@@ -91,5 +94,18 @@ func loadFlags(args []string) (*pflag.FlagSet, commandLineFlags, error) {
 	// remaining flags
 	flags.resticArgs = flagset.Args()
 
+	if flags.run != "" {
+		// abort if argument not of the right format
+		if strings.Count(flags.run, ".") < 1 {
+			fmt.Fprintf(os.Stderr, "Please provide profile name and command separated by \".\", e.g. default.check\n")
+			os.Exit(1)
```

Comment:
That would be better to return an error here, like
```go
return flagset, errors.New("please provide profile name and command separated by \".\", e.g. default.check")
```

1. it's not good practise to stop the execution of the whole program when an error arises inside a function
2. with this you can add a test to verify that a wrong argument returns an error (so we test both the _happy path_ and the _sad path_)

Reply:
Raising error and improved parsing to check for more _sad paths_.